bin/index.js

```diff
@@ -63,31 +63,27 @@ var debugLog = (log) => {
 
 // Main
 (async () => {
-
     const options = yargs
         .usage(usage)
-        .option("debug", { describe: "enable debuging logs", demandOption: false })
-        .option("p", { alias: 'port', describe: "Change default port", demandOption: false })
-        .option("ip", { describe: "Your machine public ip address", demandOption: false })
-        .option("c", { alias: 'clipboard', describe: "Share Clipboard", demandOption: false })
-        .option("w", { alias: 'on-windows-native-terminal', describe: "Enable QR-Code support for windows native terminal", demandOption: false })
-        .option("r", { alias: 'receive', describe: "Receive files", demandOption: false })
-        .option("q", { alias: 'receive-port', describe: "change receive default port", demandOption: false })
-        .option("U", { default: 'user', alias: 'username', describe: "set basic authentication username", demandOption: false })
-        .option("P", { alias: 'password', describe: "set basic authentication password", demandOption: false })
+        .option("debug", { describe: "enable debuging logs", demandOption: false, type: 'boolean' })
```

Comment:
i did’t add type for the options intentionally, there is no need for them, they just make —help output noisy. can you revert this parts?

other changes are nice 🙏

Reply:
Removed all the type from cli options.